<ui version="4.0" >
 <class>LogFunctionsWidget</class>
 <widget class="QWidget" name="LogFunctionsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Log Functions</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Flight Rules</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QTableView" name="fltRulesTable" >
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pushButton_new" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_revert" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_submit" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Aircraft Function</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QTableView" name="aircraftFunctionTable" >
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pushButton_new_2" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete_2" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_revert_2" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_submit_2" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Pilot Functions</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QTableView" name="pilotFunctionTable" >
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pushButton_new_3" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete_3" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_revert_3" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_submit_3" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="title" >
        <string>Aircraft Operation</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QTableView" name="aircraftOperationTable" >
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pushButton_new_4" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete_4" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_revert_4" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_submit_4" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../qAirLog.qrc" >
                <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qAirLog.qrc" />
 </resources>
 <connections/>
</ui>
